containers/MainContainer.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { View, Text, Modal, Button } from "react-native";
+import Nav from '../views/Nav'
+import Game from '../views/Game'
+
+export default class MainContainer extends React.Component {
+
+  constructor() {
+    super()
+    this.state = {
+      navHidden: true,
+      playerDifficulty: {
+        playerOne: 1,
+        playerTwo: 3

```

Comment:
Should we start the players at the same difficulty by default?

Reply:
Yes, I was just testing it for some reason never set it back.